src/block/divider/editor.scss

```diff
@@ -0,0 +1,136 @@
+@import "common";
+
+@include desktop {
+	:where(.has-text-align-left) .stk-block-divider {
+		--stk-dots-margin-left: 0;
+		--stk-dots-margin-right: auto;
+	}
+
+}
+
+@include desktop {
```

Comment:
You can combine these so that the CSS would be grouped together and easier to read.
```suggestion
```